Bottom: ba9c59a2feb6092a479c143a0b237324f86a38f7
Top:    f42b972505c5611ca8fdd2a2b49cfefce1cacb12
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:11:26 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 017fbf4..33fe6c3 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1295,6 +1295,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
